### PR TITLE
Fixes environment url and adds markdown syntax highlighting

### PR DESCRIPTION
**Pull request by neox5**
I also added a suffix "_ENV_VAR" for the API_TOKENs in the code examples for better understanding. 

PS: The syntax highlighting does not change the appearance in GitHub, but it helps when editing locally in vscode! ; )

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,11 +128,11 @@ If `project` depends on different projects under the same root, those are also d
...
 Multiple projects could be specified by `-p="projectA, projectB, projectC/subproject"`
 
-To deploy configuration the tool will need a valid API Token(s) for the given environments defined as environment variables - you can define the name of that env var in the environments file.
+To deploy configuration the tool will need a valid API Token(s) for the given environments defined as `environment variables` - you can define the name of that env var in the environments file.
 
 To deploy to 1 specific environment within a `environments.yaml` file, the `-specific-environment` or `-se` flag can be passed:
 
-```
+```bash
 monaco -t=environments.yaml -se=my-environment -p="my-environment" cluster
```
Comment (tatomir146) [inline README.md:136]: This also needs to be fixed
```suggestion
monaco -e=environments.yaml -se=my-environment -p="my-environment" cluster
```

**Comment by Dynatrace-Adam-Gardner** [comment]
Why not standardise on the syntax in `environments.yaml` and use `.Env` consistently?

```
my_environment:
  - name: "my_environment"
  - env-url: "{{ .Env.MY_ENVIRONMENT }}"
  - env-token-name: "{{ .Env.TOKEN_MY_ENVIRONMENT }}"
```

**Comment by didiladi** [comment]
> Why not standardise on the syntax in `environments.yaml` and use `.Env` consistently?
> 
> ```
> my_environment:
>   - name: "my_environment"
>   - env-url: "{{ .Env.MY_ENVIRONMENT }}"
>   - env-token-name: "{{ .Env.TOKEN_MY_ENVIRONMENT }}"
> ```

Because the `env-token-name` property is implemented in a way to serve as a usability improvement. The actual token is read from the environment variable as soon as something is applied to an environment and not before that. This ensures, that:
* a dry-run doesn't need any tokens
* you only need to provide the token for the environment you want to apply the configs to

E.g.: in your `environments.yaml` you defined 2 environments (dev and prod) and you just want to apply the configs to dev. To do that, you don't need to have the token for prod set as an environment variable.

@Dynatrace-Adam-Gardner I hope that makes sense?

But that is in fact a valid point - we need to add this to the documentation. After all, how should a user know this, if its not documented somewhere. I just created #51 for that.

**Review by didiladi — COMMENTED** [review]
@neox5 Cool, the changes look awesome! :+1: Can you sqash you commits once with a commit message which references #41? Thanks!

**Comment by neox5** [comment]
I'm quiet new to this game... 😄

This is my current state on my local machine:
image

My attempt would be:

1. git rebase -i 75204ce
2. mark 2 of the 3 commits with 's' to squash them down
3. assign a new commit message "Fix #41: Fixes environment-url key in README.md"
4. push the branch and now I'm stuck here:

image

If you know how to resolve this... that would be awesome 😁

**Comment by neox5** [comment]
I got it... with a little bit of help by the '+' sign 😉

`git push origin +branch`

**Review by didiladi — COMMENTED** [review]
Hi, thanks for squshing the commits! Sadly, it seems like the file `.github/ISSUE_TEMPLATE/bug_report.md` is now accidentially part of your PR. But don't worry, you can fix this by doing this:

```
git reset HEAD^ -- .github/ISSUE_TEMPLATE/bug_report.md
git commit --amend --no-edit
git push -f
```

That should remove the file from the commit and push the result.

**Comment by neox5** [comment]
Now it should be fine 😄

Thanks Dieter for the instructions 👍

**Review by didiladi — APPROVED** [review]

**Comment by didiladi** [comment]
Sure, no problem. Thank you for your contribution - this is awesome!

**Pull request merged**